File: Array/MinMax.java
import java.util.*;

class ArrayX
{
    public int Maximum(int Arr[])
    {
        int iMax = Arr[0];

        for(int iCnt=0;iCnt<Arr.length;iCnt++)
        {
            if(Arr[iCnt] > iMax)
            {
                iMax = Arr[iCnt];
            }
        }
        return iMax;
    }

    public int Minimum(int Arr[])
    {
        int iMin = Arr[0];

        for(int iCnt=0;iCnt<Arr.length;iCnt++)
        {
            if(Arr[iCnt] < iMin)
            {
                iMin = Arr[iCnt];
            }
        }
        return iMin;
    }
}
class MinMax
{
    public static void main(String Arg[])
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the size of array: ");
        int iSize = sc.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter elements of the array : ");
        for(int iCnt = 0;iCnt<iSize;iCnt++)
        {
            Arr[iCnt]=sc.nextInt();
        }

        System.out.println("Enter elements of the array : ");
        for(int iCnt = 0;iCnt<iSize;iCnt++)
        {
            System.out.print(Arr[iCnt]+" ");
        }
        System.out.println();

        ArrayX obj = new ArrayX();
        int iRet1 =  obj.Maximum(Arr);
        int iRet2 =  obj.Minimum(Arr);

        System.out.println("The maximum element is : "+iRet1);
        System.out.println("The maximum element is : "+iRet2);
    }
}

File: Array/MinMax.cpp
#include<iostream>
using namespace std;

class Array
{
    public:
        int iSize;
        int *Arr;

        Array(int iNo)//constructor
        {
            iSize = iNo;
            Arr= new int[iSize];
        }

        ~Array()//destructor
        {
            delete []Arr;   
        }

        void Accept()
        {
            cout<<"Enter the elements : "<<"\n";

            int iCnt = 0;
            for(iCnt=0;iCnt<iSize;iCnt++)
            {
                cin>>Arr[iCnt];
            }
        }

        void Display()
        {
            cout<<"Inside Display"<<"\n";

            cout<<"Elements of the array are : "<<"\n";

            int iCnt = 0;
            for(iCnt=0;iCnt<iSize;iCnt++)
            {
                cout<<Arr[iCnt]<<"\n";
            }
        }

        int Min()
        {
            int iCnt = 0;
            int iMin = Arr[0];

            for(int iCnt=0;iCnt<iSize;iCnt++)
            {
                if(Arr[iCnt] < iMin)
                {
                    iMin = Arr[iCnt];
                }
            }
            return iMin;
        }

        int Max()
        {
            int iMax = Arr[0];

            for(int iCnt=0;iCnt<iSize;iCnt++)
            {
                if(Arr[iCnt] > iMax)
                {
                    iMax = Arr[iCnt];
                }
            }
            return iMax;
        }   
};

int main()
{
    int iNo=0;
    int iRet = 0,iRet1=0;

    cout<<"Enter number of elements : "<<"\n";
    cin>>iNo;

    Array aobj(iNo);
    aobj.Accept();
    aobj.Display();

    iRet = aobj.Min();
    iRet1 = aobj.Max();

    cout<<"Minimum of all element is : "<<iRet<<"\n";
    cout<<"Maximum of all element is : "<<iRet1<<"\n";
    return 0;
}